docs/content/preview/manage/upgrade-deployment.md

```diff
@@ -78,8 +78,8 @@ Install the new version of YugabyteDB in a new directory on every YugabyteDB nod
 For example:
 
 ```sh
-wget https://downloads.yugabyte.com/yugabyte-$NEW_VER.tar.gz
-tar xf yugabyte-$NEW_VER.tar.gz -C /home/yugabyte/softwareyb-$NEW_VER/
+wget https://software.yugabyte.com/releases/{{< yb-version version="stable">}}/yugabyte-$NEW_VER-linux-x86_64.tar.gz
```

Comment:
`wget` follows redirects by default and this is also fine. 
But we have `curl` in some pages and `wget` in others.
Fine by me.